package.json

```diff
@@ -32,7 +32,7 @@
     "lint:prettier:fix": "prettier --write '**/*.{md,yaml,yml,json}'",
     "lint:styles": "stylelint 'src/**/*.scss' --report-needless-disables",
     "lint:styles:fix": "npm run lint:styles -- --fix",
-    "prepare": "husky",
+    "prepare": "npm run build",
```

Comment:
Why is this changed?

Reply:
Oh, this wasn't supposed to change; I made this modification in my own fork. It's been a while since I last worked with GitHub. I'll recreate the pull request from a separate branch.

Reply:
The changes moved here https://github.com/gravity-ui/table/pull/66/